libraries/abstractions/wifi/test/iot_test_wifi.c

```diff
@@ -648,22 +648,22 @@ TEST_GROUP_RUNNER( Full_WiFi )
     RUN_TEST_CASE( Full_WiFi, AFQP_WIFI_NetworkDelete_DeleteNonExistingNetwork );
     RUN_TEST_CASE( Full_WiFi, AFQP_WIFI_NetworkGetNonExistingNetwork );
     RUN_TEST_CASE( Full_WiFi, AFQP_WIFI_SetPMMode_InvalidPMMode );
-    RUN_TEST_CASE( Full_WiFi, AFQP_WIFI_Ping_ZeroParameters );
+    /*RUN_TEST_CASE( Full_WiFi, AFQP_WIFI_Ping_ZeroParameters ); */
```

Comment:
Please delete the commented out code here

Reply:
Deleted.